### PR TITLE
CodeGen from PR 12124 in Azure/azure-rest-api-specs - Azure HealthBot

### PR DESCRIPTION
**Pull request by guy-microsoft**
Required changes for Azure HealthBot swagger (#12124)

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
healthbot

File: src/healthbot/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Azure CLI healthbot Extension #
+This is the extension for healthbot
```
Comment (qwordy) [inline src/healthbot/README.md:2]: I suggest write more words. What is healthbot? What can it do? Who need it? Why we need it? What is typical use case?

File: src/healthbot/azext_healthbot/generated/_params.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('healthbot show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('bot_name', options_list=['--name', '-n', '--bot-name'], type=str, help='The name of the Bot '
+                   'resource.', id_part='name')
+
+    with self.argument_context('healthbot create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('bot_name', type=str, help='The name of the Bot resource.')
```
Comment (qwordy) [inline src/healthbot/azext_healthbot/generated/_params.py:34]: Add `options_list=['--name', '-n', '--bot-name']`

File: src/healthbot/azext_healthbot/generated/_params.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        c.argument('tags', tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), required=False,
+                   validator=get_default_location_from_resource_group)
+        c.argument('name', arg_type=get_enum_type(['F0', 'S1', 'C0']), help='The name of the HealthBot SKU',
```
Comment (qwordy) [inline src/healthbot/azext_healthbot/generated/_params.py:38]: `sku` is better

File: src/healthbot/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Install this extension using the below CLI command
+```
+az extension add --name healthbot
+```
+
+### Included Features ###
+#### healthbot ####
+##### Create #####
+```
+az healthbot create --bot-name "samplebotname" --location "East US" --name "F0" --resource-group "healthbotClient"
```
Comment (qwordy) [inline src/healthbot/README.md:14]: I suggest change `--bot-name` to `--name`, change `--name` to `--sku`.

**Comment by qwordy** [comment]
Please fix
```
/home/vsts/work/1/s/src/healthbot/azext_healthbot/tests/latest/example_steps.py:80:1: W391 blank line at end of file
/home/vsts/work/1/s/src/healthbot/azext_healthbot/tests/latest/test_healthbot_scenario.py:80:5: E303 too many blank lines (2)
/home/vsts/work/1/s/src/healthbot/azext_healthbot/tests/latest/test_healthbot_scenario.py:86:1: W391 blank line at end of file
1     E303 too many blank lines (2)
2     W391 blank line at end of file
```

**Comment by guy-microsoft** [comment]
Thanks @qwordy , I'm unable to push changes to my branch, so I forked the repo and added your suggestions. A new PR with the fixes is available here:
https://github.com/Azure/azure-cli-extensions/pull/2965

**Comment by qwordy** [comment]
> 
> 
> Thanks @qwordy , I'm unable to push changes to my branch, so I forked the repo and added your suggestions. A new PR with the fixes is available here:
> #2965

Let's close this one.

**Pull request closed without merging**